portality/scripts/application_status_report.py

```diff
@@ -59,27 +72,48 @@ def status_query(date_year, status):
 
     parser = argparse.ArgumentParser()
     parser.add_argument("-o", "--out", help="output file path")
-    parser.add_argument("-y", "--year", help="year to filter by")
+    parser.add_argument("-fy", "--from_year", help="from year to filter by")
+    parser.add_argument("-ty", "--to_year", help="to year to filter by")
     args = parser.parse_args()
 
     if not args.out:
         print("Please specify an output file path with the -o option")
-        parser.print_help()
-        exit()
 
-    if not args.year:
-        print("Please specify a year to filter the applications with the -y option")
+    if not args.from_year:
+        print("Please specify a 'from year' to filter the applications with the -fy option")
+
+    if not args.to_year:
+        print("Please specify a 'to year' to filter the applications with the -ty option")
+
+    if not args.out or not args.from_year or not args.to_year:
         parser.print_help()
         exit()
 
     with open(args.out, "w", encoding="utf-8") as f:
         writer = csv.writer(f)
 
-        res = models.Application.query(q=date_applied_query(args.year), size=0)
+        writer.writerow(["Applications data years", args.from_year + "-" + args.to_year])
```

Comment:
I think the task is to find the counts per year from 2018 to 2023, so shouldn't we be incrementing through and adding a row for each calendar year for the date range provided?